ui/src/app/views/concept-table/component.tsx

```diff
@@ -17,10 +19,60 @@ const styles = reactStyles({
   }
 });
 
+export class SynonymsObject extends React.Component<{},
```

Comment:
since there is no property we can have const SynonymsObject instead of class

Reply:
Resolved offline.